Emby.Server.Implementations/Services/ServiceHandler.cs

```diff
@@ -78,7 +78,8 @@ public async Task ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpRe
             var request = await CreateRequest(httpHost, httpReq, _restPath, logger).ConfigureAwait(false);
 
             httpHost.ApplyRequestFilters(httpReq, httpRes, request);
-
+            
+            httpRes.HttpContext.Items["ServiceStackRequest"] = httpReq;
```

Comment:
I would make the `"ServiceStackRequest"` string a constant.

And perhaps consider adding extension methods like `HttpContext.GetSercviceStackRequest()`  and `HttpContext.SetSercviceStackRequest(request)` to get/set it.

Reply:
Have done an extension - but like with the network change, i'm getting a circular reference as it's used between two projects.

Any ideas?

Reply:
I don't ideally want to go about duplicating the extension code per project.

Reply:
You would have to move the extension method to a different project, one that is referenced by both `Emby.Server.Implementations`, and `Emby.Dlna`.

`MediaBrowser.Common` seems like a good candidate!

Reply:
thanks.